src/rust-crypto/backup.ts

```diff
@@ -156,6 +156,9 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
             );
 
             await this.olmMachine.saveBackupDecryptionKey(backupDecryptionKey, backupCheck.backupInfo.version);
+            // Emit an event that we have a new backup decryption key, so that client can automatically
+            // start importing all keys from the backup.
```

Comment:
Isn't it up to the *library*, rather than the client, to start importing keys?

Reply:
I used sdk, it makes it more obvious than library?